src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/DebugDirectory/DebugDirectoryBuilder.cs

```diff
@@ -51,14 +53,55 @@ internal void AddEntry(DebugDirectoryEntryType type, uint version, uint stamp, i
             
             AddEntry(
                 type: DebugDirectoryEntryType.CodeView,
+                version: (portablePdbVersion == 0) ? 0 : PortablePdbVersions.DebugDirectoryEntryVersion(portablePdbVersion),
```

Comment:
Doc comments about the encoding of portablePdbVersion would be helpful in addition to making it clearer in PE spec addendum.